src/inference/include/openvino/runtime/intel_cpu/properties.hpp

```diff
@@ -63,5 +63,18 @@ static constexpr Property<bool> denormals_optimization{"CPU_DENORMALS_OPTIMIZATI
  */
 static constexpr Property<float> sparse_weights_decompression_rate{"CPU_SPARSE_WEIGHTS_DECOMPRESSION_RATE"};
 
+/**
+ * @brief This property allows to enable support of Int64 precision in the CPU plugin.
+ * @ingroup ov_runtime_cpu_prop_cpp_api
+ *
+ * By default all operations' Inputs/Outputs precisions are converted from Int64 to Int32. In some cases the range of
+ * Int32 is not enough and there is observed accuracy drop. For such cases Int64 can be used.
+ *
+ * @code
+ * core.set_property(ov::intel_cpu::enable_native_i64(true));
+ * @endcode
```

Comment:
There are some performance degradations with int64 for now, thus it is not enabled by default. Automatic switching between int64 & int32 requires blob analyzing at inference time. That will produce a big overhead.
There are just a few models for now which require int64 and this feature most likely an experimental support. If all performance issues will be resolved, that will be enabled by default and config parameter will be removed.

Reply:
Since we still perform conversion from int64 to int32, can we check during this conversion that some value are out of 32bits and if yes, then fallback to honest int64?

I understand reason of this and intention, but from user perspective it looks like a bug that CPU plugin unsafely drops range of input values. Will it drop accuracy significantly or slightly ?

BTW, should this option be integrated into `execution_mode` (see https://github.com/openvinotoolkit/openvino/pull/16367) ? 
- `performance` means enable 32bits
- `accuracy` means keep 64bits as well